packages/cardano-services/test/PgBoss/stakePoolMetadataHandler.test.ts

```diff
@@ -95,10 +97,82 @@ describe('stakePoolMetadataHandler', () => {
       expect(await metadataRepos.find({ where: { poolUpdate } })).toEqual([]);
 
       // Failing insert attempt
-      await savePoolMetadata({ dataSource, hash, metadata, poolId, poolRegistrationId });
+      await handler.savePoolMetadata({ dataSource, hash, metadata, poolId, poolRegistrationId });
 
-      // No records was inserted
+      // No records were inserted
       expect(await metadataRepos.find({ where: { poolUpdate } })).toEqual([]);
     });
   });
+  describe('attachExtendedMetadata', () => {
+    const metadata = {
+      description: 'description',
+      homepage: 'homepage',
+      name: 'name',
+      ticker: 'ticker'
+    } as Cardano.StakePoolMetadata;
+
+    it('attaches full extended metadata', async () => {
+      const expectedExtendedMetadata = DataMocks.Pool.adaPoolExtendedMetadata;
+      const attachedMetadata = handler.attachExtendedMetadata(metadata, expectedExtendedMetadata);
+
+      expect(attachedMetadata).not.toBeNull();
+      expect(attachedMetadata).toEqual({ ...metadata, ext: expectedExtendedMetadata });
+    });
+
+    it('attaches no extended metadata', async () => {
```

Comment:
I think I missed this. However, the case you suggested is not the case. I will write as below, what do you think?

`'does not modify the base metadata if URL for extended metadata does not exist in metadata'`

Reply:
How about: `does not modify the base metadata if an extended metadata URL does not exist`. I say "an" because there's the two standards for the extended metadata references.